plugins/orchestrator/store.go

```diff
@@ -29,14 +29,30 @@ type KVStore interface {
 	Reset(dataSrc string)
 }
 
-// memStore is KVStore implementation that stores data in memory.
+// KLStore describes an interface for key-label store used by dispatcher.
+type KLStore interface {
+	ListLabels(key string) Labels
+	AddLabel(key, lkey, lval string)
+	HasLabel(key, lkey string) bool
+	DeleteLabel(key, lkey string)
+	ResetLabels(key string)
+}
+
+type KStore interface {
```

Comment:
I think keeping the `K` as suffix here does not make much sense anymore. To avoid confusion might be better to just name it `Store`.

Reply:
agree